cmake/modules/FindTensorFlow.cmake

```diff
@@ -77,7 +78,18 @@ else()
   list(GET TF_INFORMATION_LIST 0 TF_DETECTED_VERSION)
   list(GET TF_INFORMATION_LIST 1 TF_DETECTED_ABI)
   list(GET TF_INFORMATION_LIST 2 TF_DETECTED_INCLUDE_DIR)
-  list(GET TF_INFORMATION_LIST 3 TF_DETECTED_LIBRARY)
+  list(GET TF_INFORMATION_LIST 3 TF_DETECTED_LIBRARY_PATH)
+
+  find_file( TF_DETECTED_LIBRARY
+    NAMES libtensorflow_framework.so.1 libtensorflow_framework.so libtensorflow_framework.dll libtensorflow_framework.dylib libtensorflow_framework.dylib.1
```

Comment:
Are we really looking for `libtensorflow_framework.so.1`? What about removing the `*.*.1` files here?